test/pummel/test-child-process-spawn-loop.js

```diff
@@ -34,17 +34,17 @@ function doSpawn(i) {
   let count = 0;
 
   child.stdout.setEncoding('ascii');
-  child.stdout.on('data', (chunk) => {
+  child.stdout.on('data', chunk => {
```

Comment:
Can you run `make lint` (or `make test`)? I think it would complain about this change…

Reply:
Fixed it